src/xunit.performance.api/XunitBenchmark.cs

```diff
@@ -52,6 +54,15 @@ static XunitBenchmark()
                     ),
                 }
             };
+
+            foreach (var pmc in PerformanceMonitorCounters)
```

Comment:
If someone wants to add a new PMC metric that is not captured by default, how do they get their data captured without it being in the list of PerformanceMonitorCounters (meaning that this is just here because we are forcing it on)?